### ISSUE TITLE
Intellisense not working in a specific scenario

### ISSUE THREAD

**Issue by DonJayamanne** [davidhalter/jedi#1025]
Can replicate in current master branch:
```python
import os
if os.path.isfile('file.txt'):
    fails = os.path.
```

No intellisense appears after the `path.`.
However if you delete the `:` after `('file.txt'):`, it works.

**Comment by davidhalter** [comment]
WTF thanks for finding this one.

**Comment by davidhalter** [comment]
Fixed in febe65f7377bdb9e1c87167e069015c4808cf55d.

I really thought long about this one and then just disabled the piece of code that was causing it. The reason for this one is a bit complicated.
